src/content/advanced/cost-optimization/cost-optimization-tools/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-linkTitle: Cost-optimization
+linkTitle: Cost-optimization-tools
```

Comment:
```suggestion
linkTitle: Tools
```

The short title (linkTitle) can omit any repitition from parent titles. It only has to work in this context.